docs/listings/panicking_test/src/lib.cairo

```diff
@@ -11,7 +11,6 @@ mod tests {
 
     #[test]
     //ANCHOR_END:first_half
-    #[should_panic(expected: 'aaa')]
```

Comment:
why this was deleted?

Reply:
shouldn't it be
#[should_panic(expected: 'panic message')]

Reply:
> why this was deleted?

[this section](https://foundry-rs.github.io/starknet-foundry/testing/testing.html#failing-tests) shows the code which should fail, but with this hidden `should_panic` it wouldn't - I think it's misleading.

> shouldn't it be
#[should_panic(expected: 'panic message')]

If we keep it, then yes, but as I said - this example should be failing on purpose.